cabal-testsuite/PackageTests/Configure/include/HsZlibConfig.h.in

```diff
@@ -3,12 +3,12 @@
 /* Define to 1 if you have the <inttypes.h> header file. */
 #undef HAVE_INTTYPES_H
 
-/* Define to 1 if you have the <memory.h> header file. */
```

Comment:
Why is this (unrelated?) file being changed by this PR?

Reply:
I have honestly no idea how this happened...
Will undo